test/rules/space-within-parens/force-two-spaces/test.ts.fix

```diff
@@ -17,6 +17,9 @@ if (  x === y  ) {
     return (  {result: true, error: (  x + y  )}  );
 }
 
-// be well
+// expected number of spaces provided
 new Foo(  );
 
+// empty parens are always allowed
+new Foo();
```

Comment:
The current implementation adds an error for `new Foo( )`, because 1 space !== 2 spaces.
If that is intended, please add a test

Reply:
Thanks for the feedback.  My understanding is that the existing test was checking for no spaces i.e. new Foo() and the corresponding error "Needs 2 whitespaces..." was appearing.  Since the empty parens should no longer error I just originally removed that error expectation and added a test to make sure 2 spaces work as expected.  There were existing tests covering the missing single and double space as well as extra spaces, so I didn't originally add more missing space checks.  Based on the review feedback I have added a test for new Foo( ); that covers 1 missing space.